src/frontends/tensorflow/src/graph_iterator_saved_model.hpp

```diff
@@ -0,0 +1,185 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <map>
+
+#include "graph_iterator_proto.hpp"
+#include "openvino/util/file_util.hpp"
+#include "saved_model.pb.h"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+struct VIBlock;
+
+template <typename T>
+std::basic_string<T> getSMName() {}
+template <typename T>
+std::basic_string<T> getVIName() {}
+
+template <>
+std::basic_string<char> getSMName<char>();
+template <>
+std::basic_string<char> getVIName<char>();
+
+#if defined(OPENVINO_ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
+template <>
+std::basic_string<wchar_t> getSMName<wchar_t>();
+template <>
+std::basic_string<wchar_t> getVIName<wchar_t>();
+#endif
+
+// Stores information about variables index
+class SMVariablesIndex {
```

Comment:
please do not shorten names

Reply:
Maybe I’ve missed it, could you point on any code style rule?

Reply:
No such document. But as a code owner, I want to have the clean code.
Come on, it is easy to fix:)

Reply:
Very friendly approach.